.github/workflows/test-open-api-spec.yml

```
@@ -0,0 +1,37 @@
+name: Validate openapi spec
+
+on:
+  push:
+    branches:
+    - features/**
+  pull_request:
+    branches:
+    - main
+    - master
+
+jobs:
+  docker:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Start containers
+      run: docker compose up -d --build
+
+    - name: Setup tests
+      run: |
+        mkdir spectral
+        # Create spectral config file
+        echo 'extends: ["spectral:oas", "https://developer.overheid.nl/static/adr/ruleset.yaml"]' > ./spectral/.spectral.yaml
+
+        docker compose exec web curl -q -o /dev/null --silent --retry-connrefused --retry 5 --retry-delay 1 --fail 'http://web:8000/openapi.yaml'  > ./spectral/openapi.yaml
```

```suggestion
        docker compose exec web curl -q -o /dev/null --silent --retry-connrefused --retry 5 --retry-delay 1 --fail 'http://web:8000/openapi.yaml'  > ./spectral/openapi.yaml
```

dankzij die `-o /dev/null` gaat dit niet werken..!
Ik had daarom in de suggestie alleen het lichtgewicht status endpoint gebruikt, dan kan je daarna de openapi call los zien.

---

Heb de twee calls nu los van elkaar. Moest hem wel met `docker compose exec web` runnen